manage.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+###!/usr/bin/env python
```

Comment:
There are too many '#' here. One is enough.

Reply:
Fixed